updater.js

```diff
@@ -0,0 +1,252 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var UNKNOWN_VERSION = 'unknown version'
+var PACKAGE_CANIUSE = 'caniuse-lite'
+
+var packageManager = ''
+
+function updateDB () {
+  var lockfile = getLockfile()
+
+  fs.readFile(lockfile, 'utf8', function (readError, lockfileRaw) {
+    if (readError) {
+      throw readError
+    }
+
+    var currentVersion = getCurrentVersion(lockfileRaw)
```

Comment:
It could be many different versions of `caniuse-lite` inside lockfile (in subdependencies). Are you sure that you are getting the minimum version here and updating all versions (including merging different versions)?